subprojects/language-java/src/main/java/org/gradle/api/tasks/javadoc/Javadoc.java

```diff
@@ -296,7 +298,7 @@ public void setClasspath(FileCollection classpath) {
      * @return The options. Never returns null.
      */
     @Nested
-    public MinimalJavadocOptions getOptions() {
+    public StandardJavadocDocletOptions getOptions() {
```

Comment:
I think we might not be able to change it because someone could have a plugin that does:
```
project.tasks.withType(Javadoc) { Javadoc javadoc ->
  javadoc.getOptions().setWindowTitle("foobar")
}
```

This would be looking for a `MinimalJavadocOptions` and now throw a `java.lang.NoSuchMethodError` now.